_drafts/2017-03-20-osd-needs-better-collaboration-tools.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  Open source design needs better collaboration tools
+date:   2017-03-09
+categories:
+  - open-source
+  - communities
+  - collaboration
+  - tools
+  - design
+---
+
+Despite the rising awareness and acceptance of UX design, in particular on the web, it has failed to gain much traction in open source software. If for argument's sake, we take UX design to have started in 1995 when [Don Norman started work for Apple as a "user experience architect"](http://blog.invisionapp.com/a-brief-history-of-user-experience/), even though it has a longer history, then the fact that design has failed to make much impact in the open source world for over 20 years suggests that there are structural and systemic barriers that make design and open source development as compatible to each other as oil and water.
+
+In this post, I will argue the need for better communication and collaboration platforms by looking at some of the fundamental characteristics of open source design and development and propose some starting points.
+
+The goal of OSD is to improve the user experience of open source software. Because of the nature of open source software development, to achieve this goal, certain critical infrastructure is necessary. I am talking about a way for designers to share and build upon ideas, interact, reach consensus and collaborate with other designers and all the other stakeholders involved in the creation of software.
+
+You may think that we already have such tools. Just look at GitHub! But when we consider the target user and use case of such tools, it becomes clear that there are no complementary equivalents for designers. Even worse, there is no tool to bridge the gap between designers and developers. GitHub and the like cater exclusively to developers' unique needs and abilities. In doing so, it excludes other stakeholders and becomes a developer silo. Design is not like this. It cannot exist in isolation. Design is about understanding, facilitation and collaboration between all the relevant stakeholders. This is why I consider collaboration tools to be so important to design. Surprisingly, the closed source world is not a great deal better off. The Atlassian and Google ecosystems are the best that I can think of, and these were not built as a platform for design. 
+
+A true collaboration platform for designers must, in some sense, bring in all stakeholders. This is where the very immature and untested idea for a collaborative design and development environment comes in. More details about this vision can be found on this [slide deck](http://slides.com/studiospring/dcd/).
+
+Because of the nature of open source software, collaboration tools are even more critical and have different needs from commercial or closed source development:
```

Comment:
I would add this earlier in the document, maybe as part the paragraph I commented on earlier.